static/app/views/issueDetails/streamline/useSuspectFlags.tsx

```diff
@@ -0,0 +1,116 @@
+import moment from 'moment-timezone';
+
+import type {Event} from 'sentry/types/event';
+import type {Organization} from 'sentry/types/organization';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import {useApiQuery} from 'sentry/utils/queryClient';
+
+type RawFlag = {
+  action: string;
+  created_at: string;
+  created_by: string;
+  created_by_type: string;
+  flag: string;
+  id: number;
+  tags: Record<string, any>;
+};
+
+export type RawFlagData = {data: RawFlag[]};
+
+export type FlagSeriesDatapoint = {
+  // flag action
+  label: {formatter: () => string};
+  // flag name
+  name: string;
+  // unix timestamp
+  xAxis: number;
+};
+
+export function hydrateFlagData({
+  rawFlagData,
+}: {
+  rawFlagData: RawFlagData | undefined;
+}): FlagSeriesDatapoint[] {
```

Comment:
```suggestion
export function hydrateFlagData(
  rawFlagData : RawFlagData | undefined;
}): FlagSeriesDatapoint[] {
```
I would also suggest renaming the function to something like toFlagSeries or something similar to indicate that you are transforming to the series type